src/ReinforcementLearningBase/src/interface.jl

```diff
@@ -19,7 +19,7 @@ import Markdown
 #####
 
 """
-    (π::AbstractPolicy)(env) -> action
+    plan!(π::AbstractPolicy, env) -> action
 
 Policy is the most basic concept in reinforcement learning. Unlike the
 definition in some other packages, here a policy is defined as a functional
```

Comment:
This isn't true anymore the above change is made.

Reply:
Very good point 🙈